index.css

```diff
@@ -0,0 +1,236 @@
+body {
+  margin: 0;
+}
+
+.font {
+  font-family: 'Poppins', sans-serif;
+}
+
+h1 {
+  color: #172b4d;
+  margin-top: 5px;
+  margin-bottom: 2px;
+}
+
+.first-section {
+  height: 552px;
+  width: 327px;
+  left: 24px;
+  top: 114px;
+  border-radius: 16px;
+  padding: 0;
+  margin-top: 43px;
+  margin-left: 31px;
+}
```

Comment:
- [ ] Following the project requirements,  The main container of the projects should use `display: grid` to lay out all the project cards.